### PR TITLE
Create breast-density-classifier.json

### PR DESCRIPTION
**Pull request by mpsampat**
@NPann adding the file for breast density classifier

### CODE REVIEW

**Review by NPann — CHANGES_REQUESTED** [review]
@mehulsampat this one required some extra care as a few key/value are left empty.

**Comment by mpsampat** [comment]
@NPann  good catch. i fixed the missing links here and also in the gitlab repo and build a new version 0.1.6

**Comment by mpsampat** [comment]
create draft and then PR to trigger build

**Review by NPann — APPROVED** [review]
LGTM

File: gears/flywheel/breast-density-classifier.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      "description": "use flag to save json containing model results."
+    },
+    "debug": {
+      "type": "boolean",
+      "default": false,
+      "description": "Enable debug mode."
+    }
+  },
+  "description": "Gear for breast density classification on mamography images",
+  "environment": {},
```
Comment (NPann) [inline gears/flywheel/breast-density-classifier.json:18]: expected that the environment key is empty ?

Comment (mpsampat) [reply gears/flywheel/breast-density-classifier.json]: @NPann sorry about that. this is fixed now.

Comment (mpsampat) [reply gears/flywheel/breast-density-classifier.json]: could you please merge after the build passes ?

**Pull request merged**